src/utils.js

```diff
@@ -112,16 +112,19 @@ var self = module.exports = {
 		}
 	},
 
-	// Merge two objects..
-	merge: (obj1, obj2) => {
-		for (var p in obj2) {
-			try {
-				if (obj2[p].constructor == Object) { obj1[p] = self.merge(obj1[p], obj2[p]); }
-				else { obj1[p] = obj2[p]; }
-			} catch(e) { obj1[p] = obj2[p]; }
-		}
-		return obj1;
-	},
+	// Merge two or more objects.
+	merge: Object.assign ||
```

Comment:
I changed this function to equal [babel's polyfill implementation](https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&code_lz=G4QwTgBCELwQ3hAZgSzAZwC4C4IHIB7AIwCsBTEAY0zwgF8BuAKCdElQDsQAbWBAOkHxUGHBABMdADQRB_EIxaUCHdAW5l-3AgHMAFJx4BKBkA&debug=false&circleciRepo=&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=6.26.0).

Generally, I noticed a couple of utility function that could be changed. We may also consider to use a dependency like lodash to ensure correctness of these functions.